tools/markdown-to-html/docs-marked-renderer.ts

```diff
@@ -49,15 +49,31 @@ export class DocsMarkdownRenderer extends Renderer {
    * Method that will be called whenever inline HTML is processed by marked. In that case,
    * we can easily transform the example comments into real HTML elements. For example:
    *
-   *  `<!-- example(name) -->` turns into `<div material-docs-example="name"></div>`
+   * `<!-- example(
+   *   {
+   *    "example": "exampleName",
+   *    "file": "example-html.html",
+   *    "lines": "[5, 10]",
+   *    "expanded": "true"
+   *   }
+   *  ) -->`
+   *  turns into
+   *  `<div material-docs-example="exampleName"
+   *        file="example-html.html"
+   *        lines="[5, 10]"
```

Comment:
```
[lines]="[5, 10]"
[expanded]="true"
```

Reply:
My bad, apparently this is just html, not angular html - so no `[` `]` around the attr names